classes/class-config.php

```diff
@@ -191,6 +194,9 @@ public function update_settings() {
 			update_site_option( 'lti_make_sites_private', 0 );
 		}
 
+		$is_helpline_url = sanitize_text_field( $_POST['is_helpline_url'] );
+		update_site_option( 'is_helpline_url', $is_helpline_url );
```

Comment:
Is the `$is_helpline_url` used only once? If so, we could get rid of the variable

```suggestion
		update_site_option( 'is_helpline_url', sanitize_text_field( $_POST['is_helpline_url'] ) );
```

Reply:
Hi Richard. This comment appears to have been missed. Could you take another look at it if you get a chance please? Many thanks!

Reply:
@richeduni please could you address this comment? Thanks :+1:

Reply:
fixed